clang/lib/Basic/Targets/AArch64.cpp

```diff
@@ -367,8 +367,21 @@ void AArch64TargetInfo::getTargetDefines(const LangOptions &Opts,
 
   // ACLE predefines. Many can only have one possible value on v8 AArch64.
   Builder.defineMacro("__ARM_ACLE", "200");
-  Builder.defineMacro("__ARM_ARCH",
-                      std::to_string(ArchInfo->Version.getMajor()));
+
+  // ACLE 5.4.1 ARM/Thumb instruction set architecture
```

Comment:
Nit: the ACLE doesn't use section numbers any more, and and the ARM/Thumb ISAs only apply to AArch32. I think this line of the comment can just be removed.